tiny_dnn/core/framework/tensor_storage.h

```diff
@@ -68,6 +68,20 @@ static inline size_t product(Container &c) {
                            std::multiplies<size_t>());
 }
 
+template <typename C1, typename C2>
+static inline size_t compute_offset(const C1 &c1, const C2 &c2) {
```

Comment:
@edgarriba You can fill new array with multiplication of last `n` elements of `c2`, reducing complexity, but that's not really needed given sizes of `c1` and `c2`.